# frame_type: 0x58(88)
# moveps: 0 -> RESUME, moveps: 1 -> PAUSE, else: ERROR
int32 moveps